JSSC + Java 9 + Windows

There seems to be a driver issue with JSSC + Java 9 + Windows
Defaults here: https://github.com/scream3r/java-simple-serial-connector/issues/116

UGS could use the patched version of `jssc.jar` in the LaserCut project which may resolve the issue in some cases:
https://github.com/wholder/LaserCut/blob/master/lib/jssc.jar

Tried replacing JSSC with [jSerialComm](http://fazecast.github.io/jSerialComm/) because the project seems more active.
It seems to work when using the classic version. However I cannot get the nbm packaging to work... 
As a side note, it lists serial ports with the prefix `cu` which could fix the issue #1016.

@winder do you have any ideas why this isn't working?
https://github.com/breiler/Universal-G-Code-Sender/commit/6e1915bb85fe03a2f7caec46b36474fd258da0ea

I get the build error: 
> [ERROR] Failed to execute goal org.codehaus.mojo:nbm-maven-plugin:4.1:manifest (default-manifest) on project ugs-platform-ugslib: Execution default-manifest of goal org.codehaus.mojo:nbm-maven-plugin:4.1:manifest failed. IllegalArgumentException -> [Help 1]
> org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.codehaus.mojo:nbm-maven-plugin:4.1:manifest (default-manifest) on project ugs-platform-ugslib: Execution default-manifest of goal org.codehaus.mojo:nbm-maven-plugin:4.1:manifest failed.

--- a/ugs-core/src/com/willwinder/universalgcodesender/utils/Settings.java
+++ b/ugs-core/src/com/willwinder/universalgcodesender/utils/Settings.java
@@ -462,7 +462,7 @@ public class Settings {
     }
 
     public ConnectionDriver getConnectionDriver() {
-        ConnectionDriver connectionDriver = ConnectionDriver.JSSC;
+        ConnectionDriver connectionDriver = ConnectionDriver.JSERIALCOMM;
         if (StringUtils.isNotEmpty(this.connectionDriver)) {
             try {
                 connectionDriver = ConnectionDriver.valueOf(this.connectionDriver);

--- a/ugs-core/test/com/willwinder/universalgcodesender/AbstractControllerTest.java
+++ b/ugs-core/test/com/willwinder/universalgcodesender/AbstractControllerTest.java
@@ -140,7 +140,7 @@ public class AbstractControllerTest {
         expect(expectLastCall()).anyTimes();
         mockMessageService.dispatchMessage(anyObject(), anyString());
         expect(expectLastCall()).anyTimes();
-        expect(mockCommunicator.openCommPort(ConnectionDriver.JSSC, port, portRate)).andReturn(true).once();
+        expect(mockCommunicator.openCommPort(ConnectionDriver.JSERIALCOMM, port, portRate)).andReturn(true).once();
         expect(instance.isCommOpen()).andReturn(false).once();
         expect(instance.isCommOpen()).andReturn(true).anyTimes();
         expect(instance.handlesAllStateChangeEvents()).andReturn(handleStateChange).anyTimes();
@@ -235,7 +235,7 @@ public class AbstractControllerTest {
         // Message for open and close.
         mockMessageService.dispatchMessage(anyObject(), anyString());
         expect(expectLastCall()).times(2);
-        expect(mockCommunicator.openCommPort(ConnectionDriver.JSSC, port, baud)).andReturn(true).once();
+        expect(mockCommunicator.openCommPort(ConnectionDriver.JSERIALCOMM, port, baud)).andReturn(true).once();
         mockCommunicator.closeCommPort();
         expect(expectLastCall()).once();
         replay(instance, mockCommunicator, mockListener);

